src/index.js

```diff
@@ -38,3 +37,19 @@ export const renderReactWithAphrodite = (name, component) => hypernova({
     return component;
   },
 });
+
+export const renderReactWithAphroditeStatic = (name, component) => hypernova({
+  server() {
+    return (props) => {
+      const { html, css } = StyleSheetServer.renderStatic(() => (
+        ReactDOMServer.renderToStaticMarkup(React.createElement(component, props))
+      ));
+
+      const style = `<style data-aphrodite>${css.content}</style>`;
```

Comment:
we should probably use `data-aphrodite="data-aphrodite"` here, for max compat with pre-HTML5. probably not a big deal tho.